xl-op/blueprint.yaml

```diff
@@ -128,7 +128,7 @@ spec:
       overrideDefault: true
       promptIf: !expr "ServerType == 'dai-release-runner' && (ProcessType == 'install' || ProcessType == 'upgrade')"
       prompt: "Enter the image tag (eg: <tagName> from <repositoryName>/<imageName>:<tagName>):"
-      default: 0.1.33
+      default: v23.3.0-409.1453-tls
```

Comment:
to check if mentioned version is okay.

Reply:
I have put the image on docker.io/xebialabsunsupported/release-remote-runner